tests/eko/evolution_operator/test_init.py

```diff
@@ -373,6 +379,7 @@ def quad_ker_pegasus(
                     10,
                     0,
                     False,
+                    "best",
```

Comment:
Yes this was supposed to be `(0,0,0,0)`, forgot to update them. 
It was not failing because all the checks on this variable are performed before calling `quad_ker`, inside `io`. Then `gamma_singlet` just need an iterable and everything that is not a number in the range `(1, n_var_ij)` will return the variation 0, which is the average.